src/lib.rs

```diff
@@ -1135,6 +1133,7 @@ mod tests {
         type Error = String;
         type Message = usize;
 
+        // The name is mentioned in tests, use a short fixed one rather than default type_name().
         fn name() -> &'static str {
             "TestActor"
         }
```

Comment:
I don't understand. Doesn't this produce the exact same name as the default impl? Note this is inside `impl Actor for 👉TestActor` block.

Reply:
> Doesn't this produce the exact same name as the default impl?

No, the default impl produces `tonari_actor::tests::TestActor` (I think I mentioned this in the description).

I'll update the comment to mention the current default value (which is documented not to be stable in https://doc.rust-lang.org/std/any/fn.type_name.html

Reply:
aha, I see!

> I think I mentioned this in the description

You did, but I wasn't reading that (I was reading individual commit messaged instead.)